### DISCUSSION TITLE
Unable to use condition rules to conditionally hide fields

### DISCUSSION THREAD

**Discussion by nathanhere** [directus/directus#11553] [Feature Requests]
(i'm not quite on the latest version of directus because it caused a startup error that's already been reported)

I am unable to use condition rules to conditionally hide fields based on the value of a many-to-one field in the same collection. This seems to work if condition rule is just using a plain text or string field, as opposed to a many-to-one field.

### To Reproduce

1. Create Collection1 with a string field "name", and add two items with name values "hi" and "bye".
2. Create Collection2 with a Many-To-One field called "words" using Collection1 for name values.
3. Create a string field "testfield" in Collection2 with a condition to be visible only when the `words -> name` equals `hi`.
4. Create new item in Collection2, and select "bye" from the "words" field.
5. Observe that the field "testfield" is still visible.

<details>
<summary>More Info</summary>

### Errors Shown

No errors shown.

### What version of Directus are you using?

9.5.0

### What version of Node.js are you using?

16.13.2

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### What operating system are you using?

MacOS

### How are you deploying Directus?

Docker Compose

</details>

**Comment by azrikahar** [comment]
Hi, it is true that currently this doesn't entirely work like your current expectation here, due to M2O fields are technically storing the `id`, and not actually something like `{ name: "hi" }` by default:

![vdGkJx4Dsr](https://user-images.githubusercontent.com/42867097/153339963-f7d1b487-b056-4826-b215-f3ed6ac2321c.gif)

Whereas it does turn into an object when we're creating a new M2O item or updating the current item. These other comments should explain what's happening here: https://github.com/directus/directus/issues/7423#issuecomment-902293310, https://github.com/directus/directus/discussions/7123#discussioncomment-1121173

And as this comment described: https://github.com/directus/directus/issues/7423#issuecomment-910918474, this does require a more thorough deliberation so I'll opt to convert this into a discussion here 👍

**Reply by Stathis90** [reply]
Are there any updates on this?

**Reply by azrikahar** [reply]
It is currently tracked in #8228 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
